README.md

```diff
@@ -71,7 +71,7 @@ The <a href="https://layer5.io/projects/service-mesh-performance">Service Mesh P
 <a href="https://layer5.io/projects/image-hub"><img src=".github/assets/images/image-hub/layer5-image-hub.png"
 style="float:left;margin:10px;" width="125px"
 alt="Image Hub" align="left" /></a>
-<a href="https://layer5.io/projects/image-hub">Image Hub</a> is a demo application which runs on <a href="https://docs.meshery.io/service-meshes/adapters/consul">Consul</a> and facilitates exploring WebAssembly modules used as Envoy Filters.
+<a href="https://layer5.io/projects/image-hub">Image Hub</a> is a demo application that runs on <a href="https://docs.meshery.io/service-meshes/adapters/consul">Consul</a> and facilitates exploring WebAssembly modules used as Envoy Filters.
```

Comment:
I don't know that there is compelling justification to switch from "which" to "that". Here's my thinking on the subject:

"That" is used to indicate a specific object, item, person, condition, etc., while "which" is used to add information to objects, items, people, situations, etc.

Reply:
Should I revert this too?

Reply:
Yes

Reply:
I just did. Please check.